documentation/book/proc-configuring-kafka-listeners.adoc

```diff
@@ -13,7 +13,7 @@
 .Procedure
 
 . Edit the `listeners` property in the `Kafka.spec.kafka` resource.
-For example:
+For example, to configure the `plain` (unencrypted) listener without authentication you would edit the `listeners` property like this:
```

Comment:
Please move it to the next line to maintain consistency across the guide and rewrite it to: An example configuration of the plain (unencrypted) listener without authentication